build/stubGenerator.php

```diff
@@ -116,7 +116,7 @@ public function getName()
 	 * @param string $name    The application name for the menu
 	 * @param array  $options An array of options to initialise the menu with
 	 *
-	 * @return  \Joomla\CMS\Menu\AbstractMenu|null  An AbstractMenu object or null if not set.
+	 * @return  \BadMethodCallException  Exception thrown as CLI Application has no menu.
```

Comment:
```suggestion
	 * @return  void
```
The function doesn't return an exception, it returns void.

Below the `@since` tag, it needs to add:
`@throws  \BadMethodCallException  Exception thrown as CLI Application has no menu.

I think that's the reason for @SharkyKZ 's thumbs down.

Reply:
Or maybe it doesn't return void, I'm not sure now. But it for sure doesn't return the exception, it throws it.

Reply:
Quite frankly I don't give a damn what the shark does - if he doesn't have enough respect to verbalise his complaints, then he should not be taken entertained at all. Any other project would have banned him completely by now.￼￼￼ he has been blocked by me so I do not see anything he does, and he has been reported to the project leadership repeatedly - but we all know this project does not take contributor safety seriously.￼￼

However I agree with you that this PR is completely stupid￼￼, despite being created by static analysis tools￼￼, and passing all of the automated tests, checks that this project provides￼

Reply:
I haven't called anything "stupid", and I always consider the possibility that I might be wrong, and it was not my intention to make you close this PR.